tools/myst/host/exec_linux.c

```diff
@@ -313,6 +343,12 @@ static int _enter_kernel(
         }
     }
 
+    // Override main stack size if present in config
```

Comment:
Does this mean the command line parameter will take effect, if the config.json does not contain a correspdoning entry? At least in "packaged mode", if the config.json does not contain main_stack_size entry, the developer's intention is to use the default stack size, which is captured in remote attestation. Commandline parameter should not be allowed to override the developer intention.

Reply:
The logic of the if condition below is the opposite: if the user has specified main_stack_size in config.json, that value will override the commandline parameter as in line 292.

However, if the user hasn't specified main_stack_size in config.json, and that user has specified the main_stack_size commandline option, that commandline option value will be effective.

The commandline parameter is supported by exec mode only. Package mode commandline doesn't recognize the main_stack_size parameter.